bedrock/firefox/templates/firefox/new/berlin/base.html

```diff
@@ -29,7 +29,7 @@
       <div class="copy">
         <img class="logo" src="{{ static('img/firefox/new/berlin/firefox-logo-wordmark.png') }}" width="408" height="103" alt="Besser digital leben — mit Firefox" />
```

Comment:
Hmm yeah... the one I have in the mockup is low res only still. I'm not going to wait on finding this as it's not a new regression, but it would be good to fix up post-merge 👍